build.ps1

```diff
@@ -31,13 +31,17 @@ param
     [ValidateSet('2017','2019')]
     [string]$vsversion ="2017",
 
+    # Build configuration - Default is Debug
     [ValidateSet('Debug', 'Release')]
-    [string]$Configuration = 'Release'
+    [string]$Configuration = 'Debug'
```

Comment:
>Debug [](start = 30, length = 5)

Why is the default debug?

Reply:
Release configuration is required only when we generated signed bits from pipeline.  For most of the development, and for people who will clone the repository I feel they would want to build in Debug configuration.  The pipeline still does everything with Release builds and the binaries which we release will be in Release Configuration